lua/tardis/sh_screens.lua

```diff
@@ -262,50 +276,181 @@ concommand.Add("tardis_toggleui", function()
 end)
 
 function TARDIS:LoadScreenUI(screen)
+	local background_img = TARDIS.visualgui_theme_basefolder
+	background_img = background_img..TARDIS:GetSetting("visual_gui_theme")
+	background_img = background_img.."/background.png"
+
 	local frame = vgui.Create("DPanel", screen)
-	frame:SetSize(screen:GetWide()-screen.gap2,screen:GetTall()-screen.gap2)
+	frame:SetSize(screen:GetWide() - screen.gap2, screen:GetTall() - screen.gap2)
 	screen.frame=frame
-	
+
 	screen.backstack={}
-	
+
+	if TARDIS:GetSetting("visual_gui_enabled")
+	then
+		frame:SetBackgroundColor(Color(0,0,0,255))
+		local frame_background=vgui.Create("DImage", frame)
+		frame_background:SetImage(background_img)
+		frame_background:SetSize( frame:GetWide(), frame:GetTall() )
+	end
+
 	local titlebar = vgui.Create("DPanel",frame)
-	titlebar:SetSize(frame:GetWide(), frame:GetTall()*0.1)
-	titlebar:SetPos(0,0)
-	titlebar:SetBackgroundColor(Color(220,220,220))
-	screen.titlebar=titlebar
-	
+	if TARDIS:GetSetting("visual_gui_enabled")
+	then
+		titlebar:SetSize(frame:GetWide(), frame:GetTall() * 0.15)
+		titlebar:SetPos(0, frame:GetTall() - titlebar:GetTall() )
+		titlebar:SetBackgroundColor(Color(1, 1, 100, 255))
+	else
+		titlebar:SetSize(frame:GetWide(), frame:GetTall() * 0.1)
+		titlebar:SetPos(0, 0)
+		titlebar:SetBackgroundColor(Color(220,220,220))
+	end
+
 	local pagename = vgui.Create("DLabel",titlebar)
-	pagename:SetTextColor(Color(0,0,0))
+	if TARDIS:GetSetting("visual_gui_enabled")
+	then
+		pagename:SetTextColor(Color(255,255,255))
+	else
+		pagename:SetTextColor(Color(0,0,0))
+	end
 	pagename:SetFont("TARDIS-PageName")
 	pagename.DoLayout = function(self)
 		pagename:SizeToContents()
-		pagename:SetPos((titlebar:GetWide()*0.5)-(pagename:GetWide()*0.5),(titlebar:GetTall()*0.5)-(pagename:GetTall()*0.5))
+		pagename:SetPos((titlebar:GetWide()*0.5)-(pagename:GetWide()*0.5),
+						(titlebar:GetTall()*0.5)-(pagename:GetTall()*0.5))
```

Comment:
Formatting is weird around this line

Reply:
GitHub's tabs are 8 spaces, that is why...
I'm used to 4, in that case, it would be perfectly aligned

![изображение](https://user-images.githubusercontent.com/32988384/108822100-a6ad7e00-75cf-11eb-9382-88701ff50a4d.png)

That's also a part of the codestyle I'm used to from working on other projects (the rule that the lines can't be too long)
If there's no place for it here, I'm reverting the change

Reply:
Yeah unfortunately the project has adopted tabs (I'd have gone spaces now for this very reason) but it's probably too late to change it. There's no real line length rule here just to be sensible but I think in this scenario it's fine to have it on one line

Reply:
Unless you want to switch to spaces (which would be a seperate issue, though very easilly solvable), this conversation has no use. Closing.